README.md

```diff
@@ -122,23 +122,24 @@ The cert-manager-csi driver aims to have complete feature parity with all
+| Attribute                                | Description                                                                                           | Default                                          | Example                          |
+|------------------------------------------|-------------------------------------------------------------------------------------------------------|--------------------------------------------------|----------------------------------|
+| `csi.cert-manager.io/issuer-name`        | The Issuer name to sign the certificate request.                                                      |                                                  | `ca-issuer`                      |
+| `csi.cert-manager.io/issuer-kind`        | The Issuer kind to sign the certificate request.                                                      | `Issuer`                                         | `ClusterIssuer`                  |
+| `csi.cert-manager.io/issuer-group`       | The group name the Issuer belongs to.                                                                 | `cert-manager.io`                                | `out.of.tree.foo`                |
+| `csi.cert-manager.io/common-name`        | Certificate common name.                                                                              |                                                  | `my-cert.foo`                    |
+| `csi.cert-manager.io/dns-names`          | DNS names the certificate will be requested for. At least a DNS Name, IP or URI name must be present. |                                                  | `a.b.foo.com,c.d.foo.com`        |
+| `csi.cert-manager.io/ip-sans`            | IP addresses the certificate will be requested for.                                                   |                                                  | `192.0.0.1,192.0.0.2`            |
+| `csi.cert-manager.io/uri-sans`           | URI names the certificate will be requested for.                                                      |                                                  | `spiffe://foo.bar.cluster.local` |
+| `csi.cert-manager.io/duration`           | Requested duration the signed certificate will be valid for.                                          | `720h`                                           | `1880h`                          |
+| `csi.cert-manager.io/is-ca`              | Mark the certificate as a certificate authority.                                                      | `false`                                          | `true`                           |
+| `csi.cert-manager.io/key-usgaes`         | Set the key usages on the certificate request.                                                        | `digital signature,key encipherment,server auth` | `server auth,client auth`        |
```

Comment:
Do we want to keep `server auth` as part of defaults given we've changed this in CM?

Reply:
This default comes from the cert-manager default. I'll update to cert-manager 0.15